### PR TITLE
unit-tests added

### PR DESCRIPTION
**Pull request by SpiderDan98**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -10,7 +10,8 @@
   ],
   "require": {
     "php": ">=7.1",
-    "fond-of-spryker/brand": "*"
+    "spryker/glue-application": "^1.0.0",
```
Comment (SpiderDan98) [inline composer.json:13]: Checked for possible versions

Comment (daniel-rose) [reply composer.json:13]: Why did you remove "fond-of-spryker/brand"? There is a client bridge to brand in glue part of this module.

**Review by daniel-rose — CHANGES_REQUESTED** [review]
@SpiderDan98 look at my comment.

File: composer.json (hunk 1 above)
Comment (SpiderDan98) [reply composer.json:13]: I just changed the version to "dev-master". Therefore Git show a remove line and an add line.

**Review by daniel-rose — APPROVED** [review]

**Pull request merged**